java/main/com/google/privacy/differentialprivacy/Count.java

```diff
@@ -97,14 +98,37 @@ public long computeResult() {
     }
 
     resultReturned = true;
-    return params
-        .noise()
-        .addNoise(
-            rawCount,
-            params.maxPartitionsContributed(),
-            params.maxContributionsPerPartition(),
-            params.epsilon(),
-            params.delta());
+    noisedCount = params
+            .noise()
+            .addNoise(
+                    rawCount,
+                    params.maxPartitionsContributed(),
+                    params.maxContributionsPerPartition(),
+                    params.epsilon(),
+                    params.delta());
+    return noisedCount;
+  }
+
+  /** ComputeConfidenceInterval computes a {@link ConfidenceInterval} with integer bounds that contains the true {@link Count} with
+   *  a probability greater or equal to 1 - alpha using the noised {@link Count} computed by {@code computeResult()}.
+   */
+  public ConfidenceInterval computeConfidenceInterval(double alpha) {
+    if (!resultReturned) {
+      throw new IllegalStateException("Noised count must be computed before calling this function.");
```

Comment:
to be consistent with Go:

"computeResult must be called before calling computeConfidenceInterval"